Source/Core/Core/IOS/ES/ES.h

```diff
@@ -284,6 +298,16 @@ class ES final : public Device
   bool LaunchPPCTitle(u64 title_id, bool skip_reload);
   static TitleContext& GetTitleContext();
 
+  // Start a title import.
+  bool InitImport(u64 title_id);
```

Comment:
Should any of these be private?  If the goal is to not misuse them then it feels like their open access is not ideal.